Subject: [PATCH] zlib: enable unaligned memory access

arm32, arm64, x86 and x86_64 are all capable of
handling unaligned memory access.

Enable UNALIGNED_OK in order to improve performance.

diff --git a/Android.bp b/Android.bp
index b4c50f5..bcec865 100644
--- a/Android.bp
+++ b/Android.bp
@@ -5,6 +5,7 @@ cc_defaults {
         "-O3",
         "-DUSE_MMAP",
         "-DZLIB_CONST",
+        "-DUNALIGNED_OK",
     ],
     stl: "none",
     export_include_dirs: ["."],
